openshift-knative-operator/cmd/operator/kodata/ingress/1.13/kourier/1-net-kourier.yaml

```diff
@@ -503,9 +486,13 @@ spec:
             allowPrivilegeEscalation: false
             readOnlyRootFilesystem: false
             runAsNonRoot: true
+            runAsUser: 65534
```

Comment:
@skonto not sure if this is on on OCP?

Reply:
A patch is missing from https://github.com/openshift-knative/net-kourier/tree/release-v1.13/openshift/patches.
See https://github.com/openshift-knative/net-kourier/tree/release-v1.12/openshift/patches. Is this intentional?

Reply:
Yes, I think we only needed it for OCP 4.10: https://github.com/openshift-knative/net-kourier/pull/122

Reply:
The user has a different effect. Did we have that patch in 1.11?

Reply:
Upps, yes copied the wrong answer here. It might be that OCP wins with its user assignment. We need to check that.